toolkits/global/packages/global-layout-with-sidebar/README.md

```diff
@@ -51,7 +51,7 @@ Somewhat unconventional, but the component can take it!
 
 ### Custom settings
 
-The `global-layout-with-sidebar` component’s settings are mapped to CSS custom properties, meaning you can alter them inline.
+The `global-layout-with-sidebar` component’s settings are mapped to CSS custom properties, meaning you can alter them inline. You can also set custom properties within your stylesheet and modify them using media queries to adjust for different view widths.
```

Comment:
I find it strange that we're encouraging overriding these values with inline styles 🤔. 
Would it be better not to mention inline at all, e.g. `meaning you can override them in your stylesheets`, and then provide example scss code?

Reply:
what is the use case for being able to override these? and agree and example would be useful

Reply:
I am also not quite sure of the use case for adjusting these styles inline.

Reply:
I think having the ability to change the value for `--with-sidebar--basis` in. your projects stylesheet to override the default value. You can also change `--with-sidebar--gap` if needed too. 

As the component is [an intrinsic layout](https://www.webdesignerdepot.com/2018/05/a-simple-introduction-to-intrinsic-web-design/) (where you [offer the browser guidance rather than strictly dictate](https://buildexcellentwebsit.es) what you want it to do) I don't think adding a media query is in the spirit of what the component is set out to do,